x-pack/plugins/security_solution/public/management/pages/policy/view/trusted_apps/empty/policy_trusted_apps_empty_unexisting.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo } from 'react';
+import { EuiEmptyPrompt, EuiButton, EuiPageTemplate } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { useGetLinkTo } from './use_policy_trusted_apps_empty_hooks';
+
+interface CommonProps {
+  policyId: string;
+  policyName: string;
+}
+
+export const PolicyTrustedAppsEmptyUnexisting = memo<CommonProps>(({ policyId, policyName }) => {
```

Comment:
Yeah, I just decided to create two different ones because the secondary button. One has a secondary button and the other one no so the behaviour is a bit different